php/Blocks/Controls/ControlAbstract.php

```diff
@@ -153,436 +148,4 @@ static function( $setting ) {
 
 		return get_object_vars( $object );
 	}
-
-	/**
-	 * Render additional settings in table rows.
-	 *
-	 * @param Field  $field The Field containing the options to render.
-	 * @param string $uid   A unique ID to used to unify the HTML name, for, and id attributes.
-	 *
-	 * @return void
-	 */
-	public function render_settings( $field, $uid ) {
```

Comment:
I should have deleted this earlier. It's from the previous editor, when it was PHP-rendered.